Launcher/Classes/Controllers/CommandsController.swift

```diff
@@ -3,6 +3,7 @@ import CommandsCore
 
 class CommandsController {
     let applicationStateHandler = ApplicationStateHandler()
+    let plistOperations = PlistOperations()
```

Comment:
Maybe we should rename PlistOperations to SettingsController/PlistHandler/PlistController? It has nothing to do with NSOperations. What do you think?

Reply:
I choose `PlistHandler` :)